kclvm/tools/src/LSP/src/test_utils.rs

```diff
@@ -0,0 +1,78 @@
+use std::{
```

Comment:
I personally believe that the components related to testing belong to the basic component and should not belong to the LSP package. Perhaps it should be placed in the `compiler_base` package.